### DISCUSSION TITLE
Purple box

### DISCUSSION THREAD

**Discussion by awildredpanda** [ppy/osu#29548] [Q&A] [answered]
Why is there a purple box in the top left of my screen?
I only just noticed it after loading into a map. 
![osu_2024-08-21_21-39-14](https://github.com/user-attachments/assets/d2e5e96b-9d01-4ed7-bd46-eb88b2ce9fa2)
![osu_2024-08-21_21-39-39](https://github.com/user-attachments/assets/b42fa1e6-a4be-459d-81c3-2ef137f4710f)
![osu_2024-08-21_21-39-58](https://github.com/user-attachments/assets/6bdc2966-acd7-4f2a-a39a-d304f287e509)

It disappeared after restarting lazer.

**Comment by peppy** [comment, marked as answer]
Discord (disable overlay).
